witwidget/notebook/colab/wit.py

```diff
@@ -186,17 +213,29 @@ def compute_custom_distance(wit_id, index, callback_name, params):
           wit.customDistanceFunctionSet = false;
         }}
       }};
-      window.updateExamplesCallback = examples => {{
+      window.updateExamplesCallback = res => {{
+        // If starting a new set of data, reset the staged examples.
+        if (res.countdown >= prevExampleCountdown) {{
+          stagedExamples = [];
+        }}
+        prevExampleCountdown = res.countdown;
+        stagedExamples.push(...res.examples);
+        if (res.countdown === 0) {{
+          // If this is the final chunk, set the staged examples.
+          window.commitUpdatedExamples();
+        }}
+      }};
+      window.commitUpdatedExamples = () => {{
         if (!wit.updateExampleContents) {{
```

Comment:
Does this work as intended? I assume it is here to prevent this function from being called before WIT frontend is ready.

Reply:
Yes in testing it is working as intended.